benchmark-ph1ex1/README.md

```diff
@@ -0,0 +1,26 @@
+Directories:
+------------
+
+* 2D-model:
+	* contains files to conduct some debugging
+	* The benchmark exercise requires a 3D model of the reactor with 232 subdomains and 26 energy group constants.
+	* The 2D-model uses 5 domains and 3 and 26 groups.
+
+* 3D-model:
+	* contains 3D geometry files
```

Comment:
also contains input files and csv results

Reply:
> The periodic vs reflective BC comparison is interesting. Have you figured out why the model with CRs out has a larger discrepancy?

The (README)[https://github.com/robfairh/mhtgr350-benchmark/blob/neutron/benchmark-ph1ex1/periodic-reflective/README.md] shows that for the periodic BCs:
* Kout = 1.0757099673061
* Kin = 1.0677552623188
And for the reflective BCs (Neumann):
* Kout = 1.0758615534905
* Kin = 1.0702136711757

So, actually, the larger discrepancy is in Kin. It makes sense, cause the control rod disturbs the flux. Then, the flux on the boundary will probably not be flat, which is the case when using reflective BCs. When the rod is out, the flux is flatter and the Neumann BC becomes a good approximation.

Reply:
Ah okay. I looked at your mesh again and I see that the geometry isn't symmetric. Yea that would definitely cause a discrepancy between periodic and reflective.